<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilinPaattaja</class>
 <widget class="QDialog" name="TilinPaattaja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tilinpäätöksen laatiminen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pic/pic.qrc">
    <normaloff>:/pic/dokumentti.png</normaloff>:/pic/dokumentti.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="otsikkoLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tilinpäätös tilikaudelle x - x</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="varoKuvake">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../pic/pic.qrc">:/pic/varoitus.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="varoLabel">
       <property name="text">
        <string>Puutteita</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="valmisteluRyhma">
     <property name="title">
      <string>Valmistavat toimet</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="poistoNappi">
          <property name="text">
           <string>Kirjaa &amp;poistot...</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/vasara.png</normaloff>:/pic/vasara.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="poistoTehty">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eiPoistettavaaLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Ei poistettavaa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="poistotKirjattuLabel">
          <property name="text">
           <string>Poistot kirjattu</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>3. Tee jaksotukset</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>4. Tee muut tarvittavat tilinpäätöskirjaukset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="jaksotusNappi">
          <property name="text">
           <string>Kirjaa &amp;jaksotukset</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/siirra.png</normaloff>:/pic/siirra.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="jaksotTehty">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eiJaksotettavaaLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Ei jaksotettavaa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="jaksotKirjattuLabel">
          <property name="text">
           <string>Jaksotukset kirjattu</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="tuloveroNappi">
          <property name="text">
           <string>Kirjaa tulovero...</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/vero.png</normaloff>:/pic/vero.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="veroTehty">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="eiveroaLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Ei verovelvollinen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="veroKirjattuLabel">
          <property name="text">
           <string>Tulovero kirjattu</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>6. Laske ja kirjaa yhteisön tulovero</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="poistoLabel">
        <property name="text">
         <string>2. Tee poistokirjaukset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>1. Varmista, että kaikki tilikaudelle kuuluvat tositteet on kirjattu</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>5. Tarkasta raportteja tutkimalla, että kaikki kirjaukset on tehty</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="paatosRyhma">
     <property name="title">
      <string>Tilinpäätös</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>7. Lukitse kirjanpito ja muodosta arkisto</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="lukitseNappi">
          <property name="text">
           <string>&amp;Lukitse kirjanpito ...</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/lukittu.png</normaloff>:/pic/lukittu.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lukitseTehty">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lukittuLabel">
          <property name="text">
           <string>Kirjanpito lukittu</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>8. Muodosta tilinpäätös ja kirjoita liitetiedot</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="tilinpaatosNappi">
        <property name="text">
         <string>&amp;Liitetiedot..</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/dokumentti.png</normaloff>:/pic/dokumentti.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>9. Tulosta tilinpäätös</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="tulostaNappi">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Tilinpäätös</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/print.png</normaloff>:/pic/print.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mappiNappi">
          <property name="text">
           <string>Kirjanpito&amp;aineisto</string>
          </property>
          <property name="icon">
           <iconset resource="../pic/pic.qrc">
            <normaloff>:/pic/arkisto64.png</normaloff>:/pic/arkisto64.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>10. Vahvista tilinpäätös</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="vahvistaNappi">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Vahvista ...</string>
        </property>
        <property name="icon">
         <iconset resource="../pic/pic.qrc">
          <normaloff>:/pic/allekirjoitus.png</normaloff>:/pic/allekirjoitus.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="ohjeNappi">
       <property name="text">
        <string>Ohje</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/ohje.png</normaloff>:/pic/ohje.png</iconset>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="suljeNappi">
       <property name="text">
        <string>Sulje</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/sulje.png</normaloff>:/pic/sulje.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>suljeNappi</sender>
   <signal>clicked()</signal>
   <receiver>TilinPaattaja</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>852</x>
     <y>662</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
